.github/workflows/copywrite.yml

```diff
@@ -1,7 +1,12 @@
 name: Check Copywrite Headers
 
 on:
-  push: {}
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - main
+      - release/**
```

Comment:
Just FYI, we have not run the copywrite tool against the release branches. We should probably though.